WordPress-VIP-Go/ruleset.xml

```diff
@@ -250,16 +236,10 @@
 	<rule ref="WordPressVIPMinimum.Variables.VariableAnalysis.UnusedVariable">
 		<severity>1</severity>
 	</rule>
-	<rule ref="WordPressVIPMinimum.UserExperience.AdminBarRemoval.HidingDetected">
-		<severity>3</severity>
-	</rule>
-	<rule ref="WordPressVIPMinimum.UserExperience.AdminBarRemoval.RemovalDetected">
+	<rule ref="WordPressVIPMinimum.UserExperience.AdminBarRemoval">
```

Comment:
Again, `severity` is a violation-level _thing_ i.e. when you call addError() or addWarning(), you can optionally set the severity there. It is not a sniff-level _thing_, so consolidation like this needs double-checking that it will still work the same as setting the severity on each violation.

Reply:
Seems to be inherited from the sniff.